src/mono/wasm/runtime/loader/blazor/_Integration.ts

```diff
@@ -73,7 +73,7 @@ export function setupModuleForBlazor(module: DotnetModuleInternal) {
         return undefined;
     };
 
-    module.downloadResource = downloadResource;
+    loaderHelpers.downloadResource = downloadResource; // polyfills were already assigned
```

Comment:
Could we drop all `Module.downloadResource` ? Also from API

Reply:
We'll remove it in API unification PR